cellxgene_schema_cli/cellxgene_schema/utils.py

```diff
@@ -11,3 +16,19 @@ def replace_ontology_term(dataframe, ontology_name, update_map):
             dataframe.loc[dataframe[column_name] == old_term, column_name] = new_term
             # remove deprecated_term from category
             dataframe[column_name] = dataframe[column_name].cat.remove_categories(old_term)
+
+
+def remove_deprecated_features(adata: ad.AnnData) -> ad.AnnData:
+    deprecated = get_deprecated_feature_ids()
+
+    # Filter out genes that don't appear in the approved annotation
+    var_to_keep = adata.var.index[~adata.var.index.isin(deprecated)].tolist()
+    adata = adata[:, var_to_keep]
+
+    # Repeat much of the same steps for the raw.var, if it exists
```

Comment:
i think this is right, but are we 100% sure we also want to remove from raw.var? Its not in the epic-level requirements, did we check with Lattice?

Reply:
I was looking at the lattice script and that is what they do. I'm not 100% sure one this. We can ask Lattice today.